src/compass/utils/lut.py

```diff
@@ -23,7 +30,19 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
     az_bistatic = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
     rg_doppler = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
 
+    if path_dem is None:
+        # Skip the FM mismatch rate calculation
+        az_fm_mismatch = None
```

Comment:
I think the check should just check if the DEM is valid. If DEM is not valid then raise an exception.